### PR TITLE
Update IO with CassandraIO

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @davorbonaci

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/beam_PreCommit_Website_Stage/521/

Jenkins built the site at commit id 577d05aa803fff82f0aef93a1e637c9c6b563a33 with Jekyll and staged it [here](http://apache-beam-website-pull-requests.storage.googleapis.com/255/index.html). Happy reviewing.

Note that any previous site has been deleted. This staged site will be automatically deleted after its TTL expires. Push any commit to the pull request branch or re-trigger the build to get it staged again.

File: src/documentation/io/built-in.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Consult the [Programming Guide I/O section]({{site.baseurl }}/documentation/prog
   <td>
     <p><a href="{{site.baseurl}}/documentation/io/built-in/hadoop/">Apache Hadoop InputFormat</a></p>
     <p><a href="https://github.com/apache/beam/tree/master/sdks/java/io/hbase">Apache HBase</a></p>
+    <p><a href="https://github.com/apache/beam/tree/master/sdks/java/io/cassandra">Apache Cassandra</a></p>
```
Comment (davorbonaci) [inline src/documentation/io/built-in.md]: sort alphabetically?

**Comment by iemejia** [comment]
@jbonofre I did @davorbonaci suggested fix and I am going to merge because I want to do a PR on top of this one so I wanted this one merged quickly.

**Pull request closed without merging**